DG-178 Fixed missing image on WS templates

cdausmus opened the pull request:
Added check for missing images on config UI
Added place holder image

To view:
 - Navigate to [Template Admin](https://digivol-test.ala.org.au/template/spotterTemplateConfig/2405386).
 - Expand Red Browed Finch - placeholder should be displayed for broken image
 - Navigate to bottom of page, click Add Animal
 - Click + to add image - placeholder should be displayed for new image
 - Click save button - dialog should be displayed saying images need to be reviewed
 - Navigate to [Filmstrip Test CM](https://digivol-test.ala.org.au/project/index/2408205) Expedition
 - Click 'Get started'
 - Placeholder image should be displayed in grid for animals with no/broken images

<img width="1190" height="502" alt="image" src="https://github.com/user-attachments/assets/d087cf34-3ebf-490d-a3c7-45fffa91e71d" />

<img width="1185" height="646" alt="image" src="https://github.com/user-attachments/assets/aeda9702-e288-4f44-aadb-6309bf0d0cff" />

cdausmus committed:
DG-178 Fixed missing image on WS templates

Added check for missing images on config UI
Added place holder image

cdausmus committed:
DG-178 Fixed small bug in config

Debugs added for checking issue in template

chrisala reviewed (commented):
The first test worked as expected.

Possibly I've misunderstood the testing notes, but in the filmstrip test (https://digivol-test.ala.org.au/transcribe/task/2408212) there were a few broken images still:
<img width="544" height="566" alt="image" src="https://github.com/user-attachments/assets/b18992a8-79d1-4f5b-81cd-2165b91d4b21" />
I couldn't see a change in the code that renders these so I suspect I have misunderstood

cdausmus committed:
DG-178 Updated WS template image display

 - Broken image displays in template config but not in transcribe
 - Image placeholder displayed in template config for new image and when image does not exist in transcribe
 - Image placeholder displayed in image detail
 - Refactored ImageController, moving key methods to new ImageService

cdausmus commented:
@chrisala Looks like I was a bit too keen.
- WS Template Config - Placeholder should display for missing/new image but broken image icon should be visible for missing images.
- WS Template Config should prevent saving while there are issues (missing images/missing hash in config defn)
- WS transcribe - Placeholder should be displayed for all missing/broken images in the grid AND in animal detail view.

I've renamed the first image for Australian Magpie in the previously mentioned template to illustrate broken image.
